TypeCobol.LanguageServer.Test/LSRTest.cs

```diff
@@ -243,6 +243,13 @@ public void ProcedureCompletionCaseInsensitive()
             LSRTestHelper.Test("ProcedureCompletionCaseInsensitive", LsrTestingOptions.NoLsrTesting, true);
         }
 
+        [TestMethod]
+        [TestCategory("Completion")]
+        public void ProcedureCompletionInputInoutOutput()
+        {
+            LSRTestHelper.Test("QualifiedNameCompletionWithFiller", LsrTestingOptions.NoLsrTesting, true);
```

Comment:
The test folder "ProcedureCompletionInputInoutOutput" is never run here. You forget to change "QualifiedNameCompletionWithFiller" by the test folder.